### ISSUE TITLE
layout block排序混乱

### ISSUE THREAD

**Issue by zhaomaoyuan** [ppaanngggg/layoutreader#17]
看到之前有个issue提到block排序较差，但是还是基本可用，但我这个非常混乱，不知道是哪里出了问题，这个是正常的吗？

<img width="660" height="688" alt="Image" src="https://github.com/user-attachments/assets/aa73d0f1-9e51-41b3-81cc-320e908585c5" />

`

                        boxes = [res['location'] for res in class_res]
                        # 将坐标归一化到0-1000范围内
                        img_h, img_w = img.shape[:2]
                        norm_boxes = []
                        for box in boxes:
                            x1, y1, x2, y2 = box
                            norm_x1 = int(x1 * 1000 / img_w)
                            norm_y1 = int(y1 * 1000 / img_h)
                            norm_x2 = int(x2 * 1000 / img_w)
                            norm_y2 = int(y2 * 1000 / img_h)
                            norm_boxes.append([norm_x1, norm_y1, norm_x2, norm_y2])

                        inputs = boxes2inputs(norm_boxes)
                        inputs = prepare_inputs(inputs, layout_reader)
                        logits = layout_reader(**inputs).logits.cpu().squeeze(0)
                        orders = parse_logits(logits, len(norm_boxes))

`

希望作者能帮我看看 感谢！

**Comment by ppaanngggg** [comment]
额, 因为这个模型不是用来排block的, 训练数据是文本行. 如果你需要排block, 建议可以用surya-order

**Comment by zhaomaoyuan** [comment]
> 额, 因为这个模型不是用来排block的, 训练数据是文本行. 如果你需要排block, 建议可以用surya-order

尝试了一下这个方法，想请教一下还有什么block level box-only的方案吗

**Comment by zhaomaoyuan** [comment]
> 额, 因为这个模型不是用来排block的, 训练数据是文本行. 如果你需要排block, 建议可以用surya-order

尝试了一下这个方法，想请教一下还有什么block level box-only的方案吗

**Comment by ppaanngggg** [comment]
你可以再看看monkeyocr，他们里面理论上有一个reading order的模型

**Comment by zhaomaoyuan** [comment]
> 你可以再看看monkeyocr，他们里面理论上有一个reading order的模型

好的，十分感谢！
